packages/mui-system/src/colorManipulator/colorManipulator.test.js

```diff
@@ -233,7 +234,7 @@ describe('utils/colorManipulator', () => {
     });
 
     it('returns a valid luminance from an CSS4 color', () => {
-      expect(getLuminance('color(display-p3 1 1 0.1)')).to.equal(0.929);
+      expect(getLuminance('color(display-p3 1 1 0.1)')).to.equal(0.928);
```

Comment:
One of the artifacts of using 8-bits channels for colors is that rounding can change a decimal here and there. This case (`display-p3`) is however somewhat complex as that colorspace has a wider gamut than sRGB (in other words, some colors can't be represented in sRGB, and that's the whole note on clipping vs gamut-mapping on a previous comment).